Documentation/contributing/release/backports.rst

```diff
@@ -152,8 +152,20 @@ Creating the backports branch
 Creating the backport pull request
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The backport pull-request may be created via the GitHub web interface, or
-alternatively you can use CLI tools to achieve these steps.
+The backport pull-request may be created via CLI tools, or alternatively
+you can use the GitHub web interface to achieve these steps.
+
+Via command-line tools
+^^^^^^^^^^^^^^^^^^^^^^
+
+These steps require all of the tools described in the :ref:`backport_setup`
+section above. It pushes the git tree, creates the pull request and updates
+the labels for the PRs that are backported, based on the ``YYYY-MM-DD.txt``
```

Comment:
Can you add the command used to generate this file? `YYYY-MM-DD.txt`

I assume it was with `./contrib/backporting/check-stable`? Ideally we should have 3 steps "Start process", "perform backports" and "submit backports"

Reply:
It's the `start-backport` script as already documented here:

http://docs.cilium.io/en/latest/contributing/release/backports/#creating-the-backports-branch

Reply:
To be clear, it's generated automatically and the docs already mention it earlier.

Reply:
Used it today, backported v1.7 so it created a `YYYY-MM-DD.txt` file. Then worked on v1.6 backport and the script overwrote the previous file. I think it'd be smart to include the backport branch or something in the filename (totally unrelated to this PR).

Reply:
ah ;) I have separate directories for each version tree to avoid messing with my main dev tree, so didn't pick this up. Good point, should be an easy tweak.

Reply:
@joestringer  See #10649